### PR TITLE
15771 - Changes to Co-op passcode formatting

### PR DESCRIPTION
**Pull request by TMunro7**
Issue #:15771
https://github.com/bcgov/entity/issues/15771

Changes to passcode formatting in the ManageBusinessDialog to follow this design:
UX design page:
https://preview.uxpin.com/c0e1953a537febcbb94df1856812c0ea4b52ccb3#/pages/161899188/simulate/sitemap

Also fixed the Staff/SbcStaff workflow to avoid the ManageBusinessDialog when adding a business following this design:
UX design page:
https://preview.uxpin.com/c0e1953a537febcbb94df1856812c0ea4b52ccb3#/pages/162795043

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue [hunk 1]
```diff
@@ -177,10 +181,20 @@ export default class SearchBusinessNameRequest extends Vue {
...
-  businessEvent (event: { name: string, identifier: string }) {
+  async businessEvent (event: { name: string, identifier: string }) {
     this.businessName = event?.name || ''
     this.businessIdentifier = event?.identifier || ''
-    this.showManageBusinessDialog = true
+    if (this.isGovStaffAccount) {
+      try {
+        let businessData: LoginPayload = { businessIdentifier: this.businessIdentifier }
+        await this.addBusiness(businessData)
+        this.$emit('add-success', this.businessIdentifier)
```
Comment (TMunro7) [inline auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue:191]: TODO: Clear out the BusinessLookup search bar after the business has been added.
image

**Comment by seeker25** [comment]
need to resync, plus pass CI

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 2]
```diff
@@ -585,7 +585,7 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   showInvalidCodeModal (label: string) {
     this.showManageBusinessDialog = false
     this.dialogTitle = `Invalid ${label}`
-    this.dialogText = `Unable to add the business. The provided ${label} is invalid.`
+    this.dialogText = `Unable to add the business. The provided ${label.toLocaleLowerCase()} is invalid.`
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:588]: toLowerCase() should be fine

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue [hunk 3]
```diff
@@ -392,7 +392,9 @@ export default defineComponent({
       nameOption.value = false
       requestAuthBusinessOption.value = false
       requestAuthRegistryOption.value = false
-      addBusinessForm.value.resetValidation()
+      if (!props.isStaffOrSbcStaff) {
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:395]: Is this needed?

File: auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue [hunk 4]
```diff
@@ -179,11 +183,24 @@ export default class SearchBusinessNameRequest extends Vue {
...
+        this.$refs.manageBusinessDialog.resetForm(true)
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.error('Error adding business: ', err)
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue:199]: Should this be wired to a dialog to show up for staff?

File: auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue (hunk 3 above)
Comment (TMunro7) [reply auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:395]: Yes, this is needed for when resetForm is called from SearchBusinesnameRequest... Without this, there is a console error when resetValidation is called and the business search bar doesn't get cleared

Comment (seeker25) [reply auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue:395]: Can you write that in a comment above it?

**Review by seeker25 — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2440-lbw1n49s.web.app

File: auth-web/package.json
Comment (Jxio) [inline auth-web/package.json:1]: missing version change on package-lock.json (did you run `npm version patch`?)

**Pull request merged**